src/core/utils/compatibility.js

```diff
@@ -61,7 +61,9 @@ export function getCompatibleVersions({
     }
 
     if (!supportsClientApp) {
-      _log.warn('addon is incompatible with clientApp', { addon, clientApp });
+      _log.warn(
+        `addon "${addon.guid}" is incompatible with clientApp: "${clientApp}"`,
```

Comment:
Maybe prefix this with `guid:"${addon.guid}"` so it's unambiguous

Reply:
yup